app/domain/authentication/authn_jwt/signing_key/fetch_cached_signing_key.rb

```diff
@@ -0,0 +1,25 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # FetchCachedSigningKey command class is a callable wrapper of FetchSigningKeyInterface interface,
+      # in order to be able to store the signing key in our cache mechanism
+      FetchCachedSigningKey ||= CommandClass.new(
```

Comment:
why do we need this class? can't we call `@fetch_signing_key_interface.fetch_signing_key` in the same place that calls this command class?

Reply:
Because provider_uri and jwks classes are not command classes and don't have call function so we need to wrap it with a command class that calls them to give it to the cache class.